test/hotspot/jtreg/compiler/jvmci/compilerToVM/DebugOutputTest.java

```diff
@@ -1,163 +0,0 @@
-/*
```

Comment:
DebugOutputTest has been removed since it's redundant with TestHotSpotJVMCIRuntime.writeDebugOutputTest.